gen.py

```diff
@@ -97,93 +101,119 @@ def convert_svg2png(infile, outfile, w, h):
 
         png_io = BytesIO()
         img.write_to_png(png_io)
-        with open(outfile, 'wb') as fout:
+        with open(outfile, "wb") as fout:
             fout.write(png_io.getvalue())
         svg.close()
         png_io.close()
         img.finish()
 
 
-# Only certain icon sizes may be covered
-try:
-    sizes = listdir("icons/{0}".format(theme))
-except FileNotFoundError:
-    exit("The theme {0} does not exists. Please reclone"
-         "the repository and try again.".format(theme))
```

Comment:
Because of the prior `try` block in line 42-43 this `try block` (previously lines 107-112) can never actually be trigger. That is, except if the theme directory is deleted between the previous check and this one. Covering that gets us into race conditions though which is a path I'd sooner not go down.